docs/en/ingest-management/elastic-agent/elastic-agent-logging.asciidoc

```diff
@@ -0,0 +1,132 @@
+[[elastic-agent-logging-config]]
+[role="xpack"]
+= Configure logging
+
+The `Logging` section of the `elastic-agent.yml` config file contains settings for configuring the logging output.
+The logging system can write logs to the `syslog`, `file`, `stderr`, `eventlog` or rotate log files.
+If you do not explicitly configure logging, the `stderr` output is used.
+
+["source","yaml",subs="attributes"]
+----
+agent.logging.level: info
+agent.logging.to_files: true
+agent.logging.files:
+  path: /var/log/elastic-agent
+  name: elastic-agent
+  keepfiles: 7
+  permissions: 0600
+----
+
+[discrete]
+[[elastic-agent-logging-settings]]
+== Logging configuration settings
+
+You can specify the following settings in the `Logging` section of the `elastic-agent.yml` config file.
```

Comment:
I wouldn't put Logging in backticks since it's not a setting name (just a section divider).